backend/api/job_board/schema.py

```diff
@@ -0,0 +1,50 @@
+from typing import List, Optional
+
+import strawberry
+
+from job_board.models import JobListing
+
+from .types import JobListing as JobListingType
+
+
+@strawberry.type
+class JobBoardQuery:
+    @strawberry.field
+    def job_listings(self, info) -> List[JobListingType]:
+        return [
+            JobListingType(
+                id=listing.id,
+                title=listing.title,
+                slug=listing.slug,
+                description=listing.description,
+                company=listing.company,
+                company_logo=(
+                    info.context.request.build_absolute_uri(listing.company_logo.url)
+                    if listing.company_logo
+                    else None
+                ),
+                apply_url=listing.apply_url,
+            )
```

Comment:
do we have a new pattern for these kind of things? are we fully migrating to DDD?

Reply:
This is what I did in another PR: https://github.com/pythonitalia/pycon/pull/2571/files#diff-d65bfc90e5a4a3392ce87e3c0eab864a0b05b1cd7ec8a860a9803a975beba1dfR65